app/lib/user_attributes.rb

```diff
@@ -0,0 +1,42 @@
+class UserAttributes
+  CONFIG_KEYS = %w[is_stored_locally].freeze
+
+  attr_reader :attributes
+
+  def initialize
+    @attributes = YAML.safe_load(File.read(Rails.root.join("config/user_attributes.yml"))).with_indifferent_access
+
+    unless Rails.env.production?
```

Comment:
Are you targeting the test env with with?
This will mean we load these for dev too won't it?